python/hail/table.py

```diff
@@ -375,14 +375,14 @@ def parallelize(cls, rows, schema, key=[], num_partitions=None):
 
     @typecheck_method(keys=oneof(str, Expression))
     def key_by(self, *keys):
-        """Change which columns are keys.
```

Comment:
Above you switched to the singluar `key`:
> The column key must be identical
and here you use the plural. I'm rather inclined to go with the singular. I would change this line to:
> Change which fields constitute the key

Reply:
Good catch, I saw and liked the switch to singular but missed the latter.

Reply:
Please don't approve if you want changes.  I just merged a different PR before noticing you'd requested a change.

Actually, I think this change was wrong: the keys are plural, they are the keys of the datasets being merged.

I did however go through Table and try to make all uses of the word key suitably singular.  Needs another look over.

Reply:
When I approved, I saw and agreed with singular key, didn't see anything else that needed changing. Dan saw it after, and I agree I should have dismissed by review then.

Reply:
I could have left my review as "requested changes" instead of the weightless "comment". When I see controversial cosmetic things like this, I am reticent to delay the PR (esp. when others have already approved).

Reply:
@danking should have requested changes with his comment, I guess.

But in the other PR where you requested a formatting change, you should have requested changes.

It's annoying if one person approves and another person requests changes, if you address changes and dismiss the review requesting changes, you can then merge, when really the person who requested changes should have to re-approve the changes.